balena-haskell-hello-world.cabal

```diff
@@ -0,0 +1,26 @@
+Name:                balena-haskell-hello-world
```

Comment:
TIL that "Case is not significant in field names". It is significant in other YAML parsers so I was surprised to see capitalisation, but according to the docs there is no issue here.